src/helpers/git.ts

```diff
@@ -0,0 +1,25 @@
+import {execSync} from 'child_process'
```

Comment:
⚠️ this file will not be included in the final PR: when #136 is merged, I'll rebase and use `simple-git` instead